backend/internal/handlers/classrooms/routes.go

```diff
@@ -27,7 +26,7 @@ func Routes(app *fiber.App, params types.Params) {
 }
 
 func classroomRoutes(router fiber.Router, service *ClassroomService) fiber.Router {
-	classroomRouter := router.Group("/classrooms").Use(middleware.Protected(service.userCfg.JWTSecret))
+	classroomRouter := router.Group("/classrooms") //.Use(middleware.Protected(service.userCfg.JWTSecret))
```

Comment:
Why are we not using protected middleware? There is no scenario where we would allow any operations on a classroom without authenticating the user.